MarsTestAutomation/Pages/ProfilePage.cs

```diff
@@ -0,0 +1,339 @@
+using System;
+using System.Threading;
+using MarsTestAutomation.Utilities;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+
+namespace MarsTestAutomation.Pages
+{
+    public class ProfilePage
+    {
+        private IWebElement descriptionField;
+        private IWebElement skillNameField;
+        private IWebElement skillLevelDropdownField;
+        private IWebElement skillLevelDropdownOption;
```

Comment:
move all the web elements to reuse for example 

    class TMPage
    {
        private IWebDriver driver;
        private IWebElement createNewButton => driver.FindElement(By.XPath("//*[@id='container']/p/a"));

        public TMPage(IWebDriver driver)
        {
            this.driver = driver;
        }

Reply:
done